analyzer/src/funTest/assets/projects/synthetic/gomod-expected-output.yml

```diff
@@ -30,7 +30,7 @@ project:
       linkage: "PROJECT_STATIC"
     - id: "GoMod::golang.org/x/sys:v0.0.0-20220610221304-9f5ed59c137d"
       linkage: "PROJECT_STATIC"
-  - name: "vendor"
+  - name: "devDependencies"
```

Comment:
I'm not sure whether this renaming is a good idea. Classically, "vendored" dependencies are dependencies that were copied to the actual project's source code. At least that's how I know the term from other languages, like Ruby. And isn't this also the case for Go? Or are these dependencies *not* copied to the project, and really retrieved from a remote, like "regular" dependencies?

Reply:
The "vendor" package can be copied to a "vendor" directory under the project root using the vendor command.
Looking at the following links, in my view it's ok to interpret it as the set of packages which actually do get copies. These are exactly the set of packages / modules needed to build and test the main module. Note that these are exactly the set of dependencies which should be included into ORT's analysis result.

Maybe "all" could be a better name to use for that "vendor" / "devDependencies" scope.
Anyhow, I suggest we just try to find good names for these two scopes. I'm open for changing these (have no strong opinion).

1. https://go.dev/ref/mod#vendoring
2. https://github.com/golang/go/blob/2a78e8afc0994f5b292bc9a5a7258c749e43032f/src/cmd/go/internal/modcmd/why.go#L26-L28